app/src/main/java/uk/nktnet/webviewkiosk/utils/createCustomWebview.kt

```diff
@@ -87,6 +89,10 @@ fun createCustomWebview(
                 mediaPlaybackRequiresUserGesture = userSettings.mediaPlaybackRequiresUserGesture
             }
 
+            if (userSettings.enableBatteryApi) {
+                addJavascriptInterface(BatteryInterface(context), "AndroidBattery")
```

Comment:
```suggestion
                addJavascriptInterface(BatteryInterface(context), "WebviewKioskBatteryInterface")
```

Let's rename this for clarity and avoid name collisions.